dash/dash-renderer/src/components/error/menu/VersionInfo.react.js

```diff
@@ -46,25 +46,20 @@ async function requestDashVersionInfo(config) {
             version: JSON.parse(cachedVersionInfo),
             link: cachedNewDashVersionLink
         };
-    } else {
-        return fetch(dashVersionUrl, {
-            method: 'POST',
-            body: JSON.stringify({
-                dash_version: currentDashVersion,
-                python_version: pythonVersion,
-                ddk_version: ddkVersion,
-                plotly_version: plotlyVersion
-            }),
-            headers: {
-                'Content-Type': 'application/json'
-            }
-        })
+    } else if (shouldRequestDashVersion(config)) {
+        const queryParams = new URLSearchParams({
+            dash_version: currentDashVersion,
+            python_version: pythonVersion,
+            ddk_version: ddkVersion,
+            plotly_version: plotlyVersion
+        }).toString();
+        return fetch(dashVersionUrl + '?' + queryParams, {mode: 'cors'})
             .then(response => response.json())
             .catch(() => {
```

Comment:
I think the catch should be moved to the end of the tail. Since it's CORS, we can't get the regular error and it will throw on the `.json()` call and go into the last then and throw again. The catch should handle all the cases.

Reply:
Also could refactor those then to `await` the fetch with a `try catch` since the function is marked as `async`